input/vocabulary/ValueSet/valueset-covid-19-condition-risk-cancer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "resourceType": "ValueSet",
+    "id": "covid-19-condition-risk-cancer",
+    "language": "en",
+    "url": "http://fhir.org/guides/acep/ValueSet/covid-19-condition-risk-cancer",
+    "version": "0.5.0",
+    "name": "COVID19RiskFactorCancer",
+    "title": "COVID-19 Risk Factor Cancer Conditions Value Set",
+    "description": "COVID-19 Risk Factor Cancer Conditions Value Set",
+    "status": "active",
+    "expansion": {
+        "identifier": "25941b18-e89a-4350-ad5c-cda8cf23fe08------",
+        "timestamp": "2021-07-19T05:50:11+10:00",
+        "total": 1,
+        "offset": 0,
+        "parameter": [
+            {
+                "name": "version",
+                "valueUri": "http://snomed.info/sct|http://snomed.info/sct/32506021000036107/version/20210204"
+            },
+            {
+                "name": "count",
+                "valueInteger": 2147483647
```

Comment:
What does 'count' represent? I saw it on the other valuesets too, so just curious!